File: 3493-maximum-number-of-operations-to-move-ones-to-the-end/maximum-number-of-operations-to-move-ones-to-the-end.py
class Solution:
    def maxOperations(self, s: str) -> int:
        one, ans = 0, 0

        for i, c in enumerate(s):
            if c == "1":
                one += 1
            elif s[i - 1] == "1":
                ans += one

        return ans

File: 3493-maximum-number-of-operations-to-move-ones-to-the-end/maximum-number-of-operations-to-move-ones-to-the-end.cs
public class Solution {
    public int MaxOperations(string s) {
        int n = s.Length;
        int one = 0, ans = 0;

        for (int i = 0; i < n; i++) {
            if (s[i] == '1') {
                one++;
            }
            else if (i > 0 && s[i - 1] == '1') {
                ans += one;
            }
        }

        return ans;
    }
}